CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [3.71.0](https://github.com/Backbase/stream-services/compare/3.70.0...3.71.0)
+### Added
+- Added new models for LE and SA to cover requirements in the new bootstrap. In the new structure, LE supports just master service agreement and custom service agreement has been dropped. Product groups, limits, job profile users, reference job roles, contacts were moved from LE to new SA model. In the new bootstrap, LE and SA are now separated and can be defined in the different JSON files and independently ingested.  
```

Comment:
Sounds good! Just a reminder that limits can be ingested at different levels, you can have SA limits, LE limits and User limits, so removing it from the LE model will create limitations on the saga.

Reply:
@uesleilima you're right, just removed the limits in the changelog. we have limits for both new SA and LE